packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -174,8 +174,8 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
         sourceFile: sourceDataResourceFile,
         projectFile: dataResourceFile,
         deployThresholdSec: {
-          onWindows: 40,
-          onOther: 30,
+          onWindows: 135,
+          onOther: 125,
```

Comment:
I left a comment here https://github.com/aws-amplify/amplify-backend/pull/1089#discussion_r1508380311 .
What do you think?

Reply:
Sorry, I missed that. Happy to add a separate test app to make this easier to track. 
Can we merge this as-is and I'll follow up with that in another PR later today or Monday?

Reply:
@edwardfoyle or the other way round. I.e. have data-with-trigger-and-other-stuff use minimal schema and have this new test explore complex schema(s) .

Reply:
> Can we merge this as-is and I'll follow up with that in another PR later today or Monday?

yes. if you can add `release-blocker` GH issue with `data-team` label on it and commit an ECD to solve it (we're at times where we want to stop adding debt).

Reply:
Created: https://github.com/aws-amplify/amplify-backend/issues/1097

Will start work on it ASAP. Let's shoot for 3/4 ECD